components/brave_rewards/browser/rewards_notification_service_impl.cc

```diff
@@ -333,7 +341,7 @@ void RewardsNotificationServiceImpl::OnGrant(RewardsService* rewards_service,
   AddNotification(notification_type,
                   args,
                   prefix + properties.promotionId,
-                  true);
+                  false);
```

Comment:
why this change?

Reply:
I don't remember exactly. I think that on Android the notification didn't show on restart as it was in a category only once for some reason.

Reply:
@gdregalo it was your change `rewards web ui relying on notifications instead of calling fetchGrants`
I suppose this change should be done for Android only?

Reply:
yes, it's for Android only. Fixed with preprocessor.